service/src/main/java/uk/gov/hmcts/reform/fpl/service/InboxLookupService.java

```diff
@@ -8,27 +9,33 @@
 import uk.gov.hmcts.reform.fpl.model.CaseData;
 import uk.gov.hmcts.reform.fpl.model.Solicitor;
 
+import java.util.ArrayList;
+import java.util.List;
 import java.util.Optional;
 
 import static uk.gov.hmcts.reform.fpl.config.LocalAuthorityEmailLookupConfiguration.LocalAuthority;
 
 @Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
 public class InboxLookupService {
     private final LocalAuthorityEmailLookupConfiguration localAuthorityEmailLookupConfiguration;
-    private final String fallbackInbox;
+    private final FeatureToggleService featureToggleService;
 
-    @Autowired
-    public InboxLookupService(LocalAuthorityEmailLookupConfiguration localAuthorityEmailLookupConfiguration,
-                              @Value("${fpl.local_authority_fallback_inbox}") String fallbackInbox) {
-        this.localAuthorityEmailLookupConfiguration = localAuthorityEmailLookupConfiguration;
-        this.fallbackInbox = fallbackInbox;
-    }
+    @Value("${fpl.local_authority_fallback_inbox}")
+    private String fallbackInbox;
 
-    public String getNotificationRecipientEmail(final CaseData caseData) {
-        return localAuthorityEmailLookupConfiguration.getLocalAuthority(caseData.getCaseLocalAuthority())
+    public List<String> getNotificationRecipientsEmails(final CaseData caseData) {
+        List<String> recipients = new ArrayList<>();
```

Comment:
Use Set instead, what if solicitor email will be same as default one? We dont want to send duplicates